common/config/jaeger.go

```diff
@@ -6,14 +6,17 @@ import (
 	"github.com/odigos-io/odigos/common"
 )
 
-var (
-	ErrorJaegerTracingDisabled = errors.New("attempting to configure Jaeger tracing, but tracing is disabled")
-	ErrorJaegerMissingURL      = errors.New("missing Jaeger JAEGER_URL config")
-	ErrorJaegerNoTls           = errors.New("jaeger destination only supports non tls connections")
+const (
+	JaegerUrlKey   = "JAEGER_URL"
+	JaegerTlsKey   = "JAEGER_TLS"
```

Comment:
The name `TLS` does not clearly describe what id does.
WDYT about calling it `JAEGER_TLS_ENABLED` which communicates it's a boolean value